spec/spec_helper.rb

```diff
@@ -10,6 +10,10 @@
     sign_in Fabricate(:user)
   end
 
+  config.before(:each, authenticated_request_user: true) do
```

Comment:
We config similar stuff in the above, it is possible to use one config for authentication? 🤔

Reply:
It is not exactly the same thing...
For system authentication (and to save time) we authenticate once for all tests using this authentication.
For requests tests, we need to re-authenticate at every single test, so I used `each` instead of `all` (using all makes requests test fail).

![image](https://user-images.githubusercontent.com/77609814/126275271-b0d4a03f-5ab9-41b5-a7b0-97f1ad5444dd.png)